install/install.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-baseurl="https://github.com/Azure/acr-docker-credential-helper/releases/download/{TRAVIS_TAG}"
+baseurl="https://aadacr.blob.core.windows.net/acr-docker-credential-helper"
 unamestr=`uname`
 
 if [[ "$unamestr" == "Linux" ]]; then
```

Comment:
This is not needed. Given the updates to the build scripts, we'll download from `baseurl` for the specific OS (Linux or Darwin). Let's sync about this in the morning.